SymbolicMathNodeData.C

```diff
@@ -38,8 +38,8 @@ RealReferenceData::D(const ValueProvider & vp)
 {
   auto rrd = dynamic_cast<const RealReferenceData *>(&vp);
 
-  std::cout << "d(" << format() << ")/d(" << vp.format()
-            << ") = " << ((rrd && &(rrd->_ref) == &_ref) ? 1 : 0) << "   (" << rrd << ")\n";
+  // std::cout << "d(" << format() << ")/d(" << vp.format()
+  //           << ") = " << ((rrd && &(rrd->_ref) == &_ref) ? 1 : 0) << "   (" << rrd << ")\n";
```

Comment:
This dynamic cast is broken for `JIT=ggcjit` builds with the clang compiler. Works fine when compiling with gcc or with any other JIT backend (even though this code is not JIT related at all!)